string msgID
string date
string time
float32 lat
string NS
string EW
string teamID
int32 sysMode
int32 AUVStatus
int32 checksum